cocos/core/scene-graph/scene-globals.ts

```diff
@@ -222,7 +222,7 @@ legacyCC.AmbientInfo = AmbientInfo;
 @help('i18n:cc.Skybox')
 export class SkyboxInfo {
     @serializable
-    protected _applyDiffuseMap = false;
+    protected _environmentLightingType = EnvironmentLightingType.Hemisphere_Diffuse;
```

Comment:
Private or protected property can use shorter names like

`_envLightingType`

B.t.w, public properties `envmap` and `environmentLightingType` doesn't follow the same naming rule, I suggest to change them to : 

- `envMap`
- `envLightingType`